plugins/woocommerce/src/Blocks/BlockTypesController.php

```diff
@@ -63,6 +63,57 @@ protected function init() { // phpcs:ignore WooCommerce.Functions.InternalInject
 		add_action( 'woocommerce_login_form_end', array( $this, 'redirect_to_field' ) );
 		add_filter( 'widget_types_to_hide_from_legacy_widget_block', array( $this, 'hide_legacy_widgets_with_block_equivalent' ) );
 		add_action( 'woocommerce_delete_product_transients', array( $this, 'delete_product_transients' ) );
+		add_filter(
+			'hooked_block_types',
+			function ( $hooked_block_types, $relative_position, $anchor_block_type, $context ) {
+				if ( $context instanceof \WP_Block_Template && 'single-product' === $context->slug ) {
+					if ( 'woocommerce/accordion-group' === $anchor_block_type && 'last_child' === $relative_position ) {
+						$hooked_block_types[] = 'woocommerce/accordion-item';
+					}
+				}
+
+				return $hooked_block_types;
+			},
+			50,
+			4
+		);
+
+		add_filter(
+			'hooked_block_woocommerce/accordion-item',
+			function (
+				$parsed_hooked_block,
+				$hooked_block_type,
+				$relative_position,
+				$parsed_anchor_block,
+				$context
+			) {
+
+				// Has the hooked block been suppressed by a previous filter?
+				if ( is_null( $parsed_hooked_block ) ) {
+					return $parsed_hooked_block;
+				}
+
+				// Only apply the updated attributes if the block is hooked after a Post Content block.
+				if ( 'woocommerce/accordion-group' === $parsed_anchor_block['blockName'] && 'last_child' === $relative_position ) {
```

Comment:
I want to check about the presence of the `__isProductDetailsChildren` to ensure that the accordion item is added in right accordion-group block.